eva-server/src/main/java/uk/ac/ebi/eva/server/ws/RegionWSServer.java

```diff
@@ -85,6 +87,22 @@ public QueryResponse getVariantsByRegion(@PathVariable("regionId") String region
         List<Region> regions = Region.parseRegions(regionId);
 
         PageRequest pageRequest = Utils.getPageRequest(queryOptions);
+        logger.info("Got page request");
```

Comment:
The logging messages in this class should be debug level, because they allow low level tracing of the application.